loleaflet/js/global.js

```diff
@@ -315,13 +315,13 @@
 			return this.decoder.decode(this.doSlice(bytes, start,end));
 		};
 		this.parseIncomingArray = function(arr) {
-//			console.debug('proxy: parse incoming array of length ' + arr.length);
+			//			console.debug('proxy: parse incoming array of length ' + arr.length);
```

Comment:
This doesn't seem right. It will look pretty weird when the `//` part is removed while debugging.